edge/pkg/devicetwin/dtcommon/util.go

```diff
@@ -13,15 +14,15 @@ func ValidateValue(valueType string, value string) error {
 	case "":
 		valueType = "string"
 		return nil
-	case "string":
+	case "string", "bytes":
```

Comment:
Have any consts for these type?

Reply:
Yes, there should be.
As defined in device crd, the types include "int, string, double, float, boolean, bytes".